package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hmcts/ccd-case-ui-toolkit",
-  "version": "2.70.0-fix-alert-service",
+  "version": "2.70.1-alert-service-update",
```

Comment:
As a minimum, this is a minor release, not a patch. It should go up to `2.71.0`.

Reply:
Agreed

Reply:
Scrub that, this includes a breaking change. It should be version `3.0.0`.